### PR TITLE
update paper

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
Here some changes suggested in #56 

Also, I've moved the paper to a paper folder so it there is fewer files at the top level of the repo.

@strengejacke @pdwaggoner tell me what you think

### CODE REVIEW

**Comment by strengejacke** [comment]
I like the idea with the figure that describes the different model parts and how the functions are related.

According the order of authors, that's something you both should decide.

**Comment by strengejacke** [comment]
> Also, I've moved the paper to a paper folder so it there is fewer files at the top level of the repo

Can you update the rebuildignore accordingly?

**Comment by strengejacke** [comment]
From my point of view, ready to merge. Let's wait for the OK from @pdwaggoner

**Comment by DominiqueMakowski** [comment]
yup! Fun fact, there is exactly 12 hours of difference between me and @pdwaggoner and 6 with @strengejacke, we are covering half the planet 😃

**Comment by strengejacke** [comment]
> Fun fact

Yeah, I was thinking about the same thing. ;-) Pro: 24h-working, Con: Giving feedback sometimes takes a bit longer, because 1-2 people are sleeping.

**Comment by pdwaggoner** [comment]
Apologies for my delay - the fun fact is surely the reason. I was sound asleep during these commits! But this all looks great to me. Thanks again for the work on this guys. Very exciting!

File: paper/paper.md [hunk 1]
```diff
@@ -8,32 +8,36 @@ authors:
...
 
 For a more intuitive introduction, consider the following examples for three major types of models: ordinary least squares (OLS) regression, linear mixed effects models, and Bayesian models. Importantly, though only a few functions are included below for the sake of space, users are encouraged to inspect the package documentation for an exhaustive list of package functionality with accompanying examples.
 
+![The functions in insight allow to access different aspects of models, such as the data used for fitting, the parameters of the fitted model or various information about the model.](figure2.png)
```
Comment (pdwaggoner) [inline paper/paper.md]: Did you want to italicize the package name each time (*insight*)? Or should we leave it standard font? I don't have a preference here; just wanting to be consistent for the manuscript.

File: paper/paper.md [hunk 2]
```diff
@@ -119,9 +123,9 @@ find_formula(sample3)
...
+*insight* works with many different model-objects, such as **AER** (*ivreg, tobit*), **afex** (*mixed*), **base** (*aov, aovlist, lm, glm*), **betareg** (*betareg*), **blme** (*blmer, bglmer*), **brms** (*brmsfit*), **crch**, **countreg** (*zerontrunc*), **coxme**, **estimatr** (*lm_robust, iv_robust*), **feisr** (*feis*), **gam** (*Gam*), **gamm4** , **gamlss**, **gbm**, **gee**, **geepack** (*geeglm*), **GLMMadaptive** (*MixMod*), **glmmTMB** (*glmmTMB*), **gmnl**, **lfe** (*felm*), **lme4** (*lmer, glmer, nlmer, glmer.nb*), **MASS** (*glmmPQL, polr*), **mgcv** (*gam, gamm*), **nnet** (*multinom*), **nlme** (*lme, gls*), **ordinal** (*clm, clm2, clmm*), **plm**, **pscl** (*zeroinf, hurdle*), **quantreg** (*rq, crq, rqss*), **robust** (*glmRob, lmRob*), **robustbase** (*glmrob, lmrob*), **robustlmm** (*rlmer*), **rstanarm** (*stanreg, stanmvreg*), **survey**, **survival** (*coxph, survreg*), **truncreg** (*truncreg*), **VGAM** (*vgam, vglm*) and more.
```
Comment (pdwaggoner) [inline paper/paper.md]: should insert a comma after "**VGAM** (*vgam, vglm*)" before "and more."

File: paper/paper.md (hunk 1 above)
Comment (strengejacke) [reply paper/paper.md]: This is a tooltip, I think, so it's not necessary here... But we can check the generated PDF later.

File: paper/paper.md (hunk 2 above)
Comment (strengejacke) [reply paper/paper.md]: thanks, done!

**Pull request merged**